modules/rosa-create-objects.adoc

```diff
@@ -0,0 +1,479 @@
+:_module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-manage-objects-cli.adoc
+
+[id="rosa-create-objects_{context}"]
+= Create objects
+
+[role="_abstract"]
+This section describes the `create` commands for clusters and resources.
+
+[id="rosa-create-admin_{context}"]
+== create admin
+
+Creates a cluster administrator with an automatically generated password that can log in to a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create admin --cluster=<cluster_name>
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster to add to the identity provider (IDP).
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+|===
+
+.Example
+Create a cluster administrator that can log into a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa create admin --cluster-name=mycluster
+----
+
+[id="rosa-create-cluster_{context}"]
+== create cluster
+
+Creates a new cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create cluster --cluster=<cluster_name> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster. This is used when generating a sub-domain for your cluster on `openshiftapps.com`.
+
+|--channel-group
+|The channel group (string) is the name of the group where this image belongs, for example `stable` or `fast`. Default: `stable`
+
+|--compute-machine-type
+|The instance type (string) for the compute nodes. Determines the amount of memory and vCPU allocated to each compute node.
+
+|--compute-nodes
+|The number (integer) of worker nodes to provision per zone. Single zone clusters require at least 2 nodes. Multi-zone clusters require at least 3 nodes. Default: `2` for single-az; `3` for multi-az
+
+|--disable-scp-checks
+|Indicates whether cloud permission checks are disabled when attempting the installation of the cluster.
+
+|--dry-run
+|Simulates creating the cluster.
+
+|--enable-autoscaling
+|Enable autoscaling of compute nodes. By default, autoscaling is set to `2` nodes. To set non-default node limits, use this argument with the `--min-replicas` and `--max-replicas` arguments.
+
+|--host-prefix
+|The subnet prefix length (integer) to assign to each individual node. For example, if host prefix is set to `23`, then each node is assigned a `/23` subnet out of the given CIDR.
+
+|--machine-cidr
+|Block of IP addresses (ipNet) used by OpenShift while installing the cluster. Example: `10.0.0.0/16`
```

Comment:
s/OpenShift/{product-title} ?

Reply:
Changed to OpenShift Container Platform. Not implementing {product-title} for this release.